medical/models/medical_patient.py

```diff
@@ -107,3 +108,11 @@ def _get_default_image_path(self, vals):
         return get_module_resource(
             'medical', 'static/src/img', 'patient-avatar.png',
         )
+
+    def _search_age(self, operator, value):
+        current_date = date.today()
+        first_possible_birthdate = (current_date -
+                                    relativedelta(years=int(value)+1))
+        possible_dates = [first_possible_birthdate + relativedelta(days=x)
```

Comment:
What date would I compare with the patient's birthdate? If the user enters the patient's age, aren't there 365 dates that match up with the age (presuming they're just entering the number of years old the patient is and not also the number of months and days)?